pages/index.js

```diff
@@ -10,13 +9,18 @@ export default function HomePage({
   onCreateEmotion,
   onToggleBookmark,
   myBookmarkedEmotions,
+  emotionTypes,
 }) {
   const [selectedFilterButton, setSelectedFilterButton] = useState("");
 
   return (
     <>
-      <StyledHeading>Mood Wave</StyledHeading>
-      <EmotionForm emotions={emotions} onSubmit={onCreateEmotion} />
+      <Heading>Mood Wave</Heading>
+      <EmotionForm
+        emotions={emotions}
```

Comment:
your emotionForm does not accept a prop called `emotions`, but is expecting an onCancel event handler.

Reply:
You are right. I deleted the emotions prop on the indexpage.
But the onCancel prop is coming from the edit.js. I have to leave it here.